pdp_util/ensemble_members.py

```diff
@@ -26,6 +26,11 @@ def __call__(self, environ, start_response):
             ensemble = sesh.query(Ensemble).filter(Ensemble.name == ensemble_name).first()
 
             if not ensemble: # Result does not contain any row therefore ensemble does not exist
+                # TODO: Instead of returning a text/plain response, this should
+                #  return a json response with an empty array, consistent with
+                #  the datatype for other ensemble names. OR, possibly better,
+                #  it should return a 404 not found -- because, like, ensemble
+                #  not found. But god only knows what depends on this choice.
```

Comment:
Good, then I'd like to change this. I am leaning towards a 404 Not Found. And in that case, although it would be a one-off, should we return JSON body with an informative message? That is common practice in web APIs these days. Possibly this should be saved for a broader effort that updates all 404's, but I have a shrewd idea of when that will actually happen.

Reply:
Done